wiki/Contests/en.md

```diff
@@ -2,291 +2,418 @@
+| Vocal² | ![osu!][osu!] | ![][flag_KR] [Heilia](https://osu.ppy.sh/users/9823042) | ![][flag_DE] [jamesjan3](https://osu.ppy.sh/users/6260705) | ![][flag_CN] [Garden](https://osu.ppy.sh/users/2849992) | [Results](https://osu.ppy.sh/community/forums/topics/1198663) |
+| Instrumental³ | ![osu!][osu!] | ![][flag_RU] [xbopost](https://osu.ppy.sh/users/6842421) | ![][flag_ID] [\[Keqing\]](https://osu.ppy.sh/users/8972308) | ![][flag_KR] [Acylica](https://osu.ppy.sh/users/1943309) | [Results](https://osu.ppy.sh/community/forums/topics/1198663) |
 
-- [Contest thread](https://osu.ppy.sh/community/forums/topics/415669)
-- [Results post](https://osu.ppy.sh/community/forums/topics/432070)
+¹ The Featured Artist category provides mappers with a choice of two different songs to be mapped: Zekk - Re\_construct and Shuu no Hazama \[Rainshower\].
 
-#### Newspaper Cup \#3 (2017)
+² The Vocal category provides mappers with a choice of two different songs to be mapped: nonoc - memento and DUSTCELL - DERO.
 
-| Songs No. | Songs | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] |
-| :-- | :-- | :-- | :-- |
-| 1 | 影翔鼓舞 - Oriental Blossom | [yf_bmp](https://osu.ppy.sh/users/1243669), [EmingK](https://osu.ppy.sh/users/2021118) | [Gaia](https://osu.ppy.sh/users/2683648), [Doj](https://osu.ppy.sh/users/4121612) |
-| 2 | ESQUARIA - 寒椿 ～ Kantsubaki | [Regou](https://osu.ppy.sh/users/419954), [fish39](https://osu.ppy.sh/users/3522390) | [Bluekrait](https://osu.ppy.sh/users/4205741), [Rose Pacifica](https://osu.ppy.sh/users/1393255) |
+³ The Instrumental category provides mappers with a choice of two different songs to be mapped: onoken - Felys final remix and Nekomata Master - The Sky of Sadness.
 
-- [Contest thread](https://osu.ppy.sh/community/forums/topics/546038)
-- [Results post](https://osu.ppy.sh/community/forums/topics/570350)
-
-#### Newspaper Cup \#4 (2017)
-
-| Songs No. | Songs | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] |
-| :-- | :-- | :-- | :-- |
-| 1 | 葉月ゆら - サラマンドラの踊り子 | [Yugu](https://osu.ppy.sh/users/Yugu) | [nika mika](https://osu.ppy.sh/users/nika%20mika) |
-| 2 | P＊Light - SAY BAY | [Firika](https://osu.ppy.sh/users/Firika) | [Yugu](https://osu.ppy.sh/users/Yugu) |
+### Newspaper Cup
 
-- [Contest thread](https://osu.ppy.sh/community/forums/topics/690824)
-- [Results post](https://osu.ppy.sh/community/forums/topics/721264)
+The *Newspaper Cup* is a beatmapping contest that is similar to the Pending Cup in nature but with a much greater emphasis on novice mappers. As with Pending Cup, the contest is also being held yearly by prominent members of the Chinese mapping community and has since been developed from a contest that was particularly intended for the Chinese-speaking members of the community to a global-scale contest.
```

Comment:
```suggestion
The **Newspaper Cup** is a mapping contest that is similar to the Pending Cup in nature but with a much greater emphasis on novice mappers. As with Pending Cup, the contest is also being held yearly by prominent members of the Chinese mapping community and has since been developed from a contest that was particularly intended for the Chinese-speaking members of the community to a global-scale contest.
```